### ISSUE TITLE
How to inject

### ISSUE THREAD

**Issue by VVVANNONVVVV** [Broihon/GH-Injector-Library#1]
Hello.
Can you tell me how to inject some dll into some process with that library ?

**Comment by GH-Rake** [comment]
Call one of these functions:

`DWORD _LoadLibraryExW	(const wchar_t * szDllFile, HANDLE hTargetProc, LAUNCH_METHOD Method, DWORD Flags, HINSTANCE & hOut, DWORD & LastWin32Error);
DWORD _LdrLoadDll		(const wchar_t * szDllFile, HANDLE hTargetProc, LAUNCH_METHOD Method, DWORD Flags, HINSTANCE & hOut, DWORD & LastWin32Error);
DWORD _LdrpLoadDll		(const wchar_t * szDllFile, HANDLE hTargetProc, LAUNCH_METHOD Method, DWORD Flags, HINSTANCE & hOut, DWORD & LastWin32Error);
DWORD _ManualMap		(const wchar_t * szDllFile, HANDLE hTargetProc, LAUNCH_METHOD Method, DWORD Flags, HINSTANCE & hOut, DWORD & LastWin32Error);
`

these declarations can be found here: https://github.com/Broihon/GH-Injector-Library/blob/2365cd332bb39c04b46e3ff037e9246d1476d54e/GH%20Injector%20Library/Injection.h#L84

**Comment by Broihon** [comment]
> 
> 
> Hello.
> Can you tell me how to inject some dll into some process with that library ?
The readme provides all the info you need. Check the provided Injection.h for more information.
Tl;dr: compile the library, import it with LoadLibrary, use GetProcAddress.
